openedx_caliper_tracking/processor.py

```diff
@@ -24,7 +26,8 @@ def log_success(event_id, status_code):
     event_id: (str) UUID of the caliper event
     status_code: (int) HTTP status code of the repsonse from the API
     """
-    LOGGER.info('Success {}: Caliper event delivery successful for event id: {} to endpoint: {}'.format(
+    LOGGER.info(('Success {}: Caliper event delivery '
+                'successful for event id: {} to endpoint: {}').format(
         status_code,
         event_id,
         settings.CALIPER_DELIVERY_ENDPOINT
```

Comment:
I think indentation need to be fixed here?

Reply:
No it is compliant with flake8.